Musico/app/views/songs/_song_users.html.erb

```diff
@@ -0,0 +1,19 @@
+<% @users.each do |user| %>
+    <div class="checkbox_div">
+        <%= check_box_tag "song[users][]" , user.id, false, label: user.name, id: :search_user_checkbox %>
```

Comment:
You should not use same id for multiple elements in the same page

Reply:
done